MAUI/ListView/item-drag-and-drop.md

```diff
@@ -0,0 +1,485 @@
+---
+layout: post
+title: Item Reordering in .NET MAUI ListView control | Syncfusion
+description: Learn here all about Item Reordering support in Syncfusion .NET MAUI ListView (SfListView) control and more.
+platform: MAUI
+control: SfListView
+documentation: ug
+---
+
+# Item Reordering in .NET MAUI ListView (SfListView)
+
+The `SfListView` allows reordering by dragging and dropping items. It supports displaying the customized view in a template while dragging the item. It can be enabled by setting the `SfListView.DragStartMode` property to `OnHold`. The drag and drop options are listed as follows:
+
+ * `None`: Disables drag and drop. This is the default value.
+ * `OnHold`: Allows dragging and dropping by holding the item.
+ * `OnDragIndicator`: Allows dragging and dropping by loading the `DragIndicatorView` within `SfListView.ItemTemplate`.
+
+N> The `GridLayout` does not support drag and drop. 
+
+The drag and drop scenarios are as follows:
+
+ * Items can be reordered to any position by auto-scrolling.
+ * Items can be reordered in same group or in other groups but, no groups can be added to other groups.
+ * Groups, header, and footer cannot be reordered.
+
+To enable drag and drop using 'OnHold', follow the code example:
+
+{% tabs %}
+{% highlight xaml %}
+<ContentPage  xmlns:syncfusion="clr-namespace:Syncfusion.Maui.ListView;assembly=Syncfusion.Maui.ListView">
+  <syncfusion:SfListView x:Name="listView" 
+                   ItemsSource="{Binding ToDoList}"
+                   DragStartMode="OnHold"
+                   BackgroundColor="#FFE8E8EC"
+                   ItemSize="60" />
+</ContentPage>
+{% endhighlight %}
+{% highlight c# %}
+listView.DragStartMode = DragStartMode.OnHold; 
+{% endhighlight %}
+{% endtabs %}
+
+To enable drag and drop using both 'OnHold' and 'OnDragIndicator', follow the code example:
```

Comment:
To enable drag and drop using both 'OnHold' and 'OnDragIndicator', follow the code example below.

Reply:
Addressed the concerns provided.